src/pybind/mgr/prometheus/module.py

```diff
@@ -298,6 +298,19 @@ class Module(MgrModule):
             name='rbd_stats_pools_refresh_interval',
             type='int',
             default=300
+        ),
+        Option(
+            name='standby_behaviour',
+            type='str',
+            default='default',
+            enum_allowed=['default', 'error']
+        ),
+        Option(
+            name='standby_error_status_code',
+            type='int',
+            default=500,
+            min=400,
+            max=599
```

Comment:
As modifying both settings has an immediate effect (no restar required), let's flag them with runtime:
```suggestion
        Option(
            name='standby_behaviour',
            type='str',
            default='default',
            enum_allowed=['default', 'error'],
            runtime=True
        ),
        Option(
            name='standby_error_status_code',
            type='int',
            default=500,
            min=400,
            max=599,
            runtime=True
```

Reply:
One thing I came across while testing: changing the config value has immediate effect, but deleting the config key does not ensure a fallback to the default value? If standby_behaviour is set to 'error' and the config key is deleted, I would expect the 'default'-behaviour again.